Source/Blazorise/Components/Droppable/DropZone.razor.cs

```diff
@@ -212,14 +323,14 @@ private async Task OnDropHandler()
 
         private IEnumerable<TItem> GetItems()
         {
-            Func<TItem, bool> predicate = ( item ) => ParentContainer.ItemsFilter( item, Name );
+            Func<TItem, bool> predicate = ( item ) => ParentContainer.ItemsFilter( item, Name ?? string.Empty );
 
             if ( ItemsFilter != null )
             {
                 predicate = ItemsFilter;
             }
 
-            return ( ParentContainer?.Items ?? Enumerable.Empty<TItem>() ).Where( predicate ).ToArray();
+            return ( ParentContainer?.Items ?? Enumerable.Empty<TItem>() ).Where( predicate ).OrderBy( x => GetItemIndex( x ) ).ToArray();
```

Comment:
Are we still using the index functionality if `AllowReorder` is false? Does this make it work differently?

Reply:
It will work the same I think.